tests/integration/components/popup-test.js

```diff
@@ -142,3 +142,16 @@ test('popup closes when layer is destroyed', function(assert) {
   assert.equal(map._popup, null, 'popup closed');
 
 });
+
+test('popupOptions hash', function(assert) {
+  this.set('markerCenter', locations.nyc);
+  this.render(hbs`
+    {{#leaflet-map zoom=zoom center=center}}
+      {{#marker-layer location=markerCenter draggable=draggable popupOptions=(hash className="foo")}}
```

Comment:
similar thing in this test: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Fintegration%2Fcomponents%2Fleaflet-map-test.js#L86-L98